app/views/articles/_share_on_social_networks.html.erb

```diff
@@ -1,14 +1,25 @@
+<%
+twitter_link  = article.draft? ? '#' : "https://twitter.com/intent/tweet?text=" \
+                                         "#{url_encode(article.title)}&url=#{article_url}&via=crimethinc"
+facebook_link = article.draft? ? '#' : "https://www.facebook.com/dialog/feed?" \
+                                         "app_id=966242223397117&display=popup&caption=" \
+                                         "#{url_encode(article.title)}&link=#{article_url}" \
+                                         "&redirect_uri=http://www.facebook.com/&display=popup"
+tumblr_link = article.draft? ? '#' : "http://tumblr.com/widgets/share/tool?canonicalUrl" \
+                                       "=#{article_url}&caption=#{url_encode(article.title)}" \
+                                       "&content=#{article.image}"
```

Comment:
Instead of this, can you wrap this whole partial in an `if article.published?` ?

Reply:
will do